include/seastar/core/metrics.hh

```diff
@@ -297,26 +297,18 @@ public:
         return std::get<double>(u);
     }
 
+    /// Returns an unsigned uint64_t value rounded and trimmed to
+    /// [0,9223372036854775807]
```

Comment:
This is very odd, if it's a uint64_t, why limit to 2^63-1 and not 2^64-1?
I think this deserves a comment, if not a fix. The sentence below "note that the maximum value fits in signed int64_t" doesn't explain what is going on here, and why.

Reply:
The reason is that lround() returns (signed) long and there's no library function for unsigned conversion.
I'll add that to the comment